prow/config/config.go

```diff
@@ -2223,21 +2223,46 @@ func (c *Config) ValidateJobConfig() error {
 	// Set the interval on the periodic jobs. It doesn't make sense to do this
 	// for child jobs.
 	for j, p := range c.Periodics {
-		if p.Cron != "" && p.Interval != "" {
-			errs = append(errs, fmt.Errorf("cron and interval cannot be both set in periodic %s", p.Name))
-		} else if p.Cron == "" && p.Interval == "" {
-			errs = append(errs, fmt.Errorf("cron and interval cannot be both empty in periodic %s", p.Name))
-		} else if p.Cron != "" {
+		seen := 0
```

Comment:
will need unit test to help better understand this

Reply:
Done with https://github.com/kubernetes/test-infra/pull/26743/commits/1fd0663d9066501e2ca63cf48d0e4eac7b72f734.